src/compiler.ts

```diff
@@ -99,6 +99,17 @@ const getFlags = (language: Language, srcPath: string): string[] => {
             ret = [srcPath, '-d', binDir, ...args];
             break;
         }
+        case 'hs': {
+            ret = [
+                srcPath,
+                '-o',
+                getBinSaveLocation(srcPath),
+                '-outputdir',
+                '/tmp/ghc/',
```

Comment:
This line won't work for Windows / other OSes without /tmp.
Also, what happens if ghc subdir doesn't exist?

Reply:
ghc will handle the non-existent folder, but you are right, I forgot about Windows.
I found some [better flags](https://downloads.haskell.org/ghc/latest/docs/users_guide/separate_compilation.html#keeping-intermediates) that also will leave the working directory unchanged after running testcases.